javascript/Spread_operator.md

```diff
@@ -164,8 +152,11 @@ for (let i = 0; i < arr1.length; i++) {
 }
 }
 ```
+[from freecodecamp](https://www.freecodecamp.org/forum/t/how-do-i-build-a-nested-object-in-javascript-dynamically/304543/3)
+
 ## Additional Materials
 
-- [Spread-Operator Quiz](https://javascript.info/rest-parameters-spread)
+- [Spread-Operator Quiz](https://www.codingame.com/playgrounds/7998/es6-tutorials-spread-operator-with-fun)
```

Comment:
This link again is not a quiz, it's rather a playground. So either change the label to playground or change the link